test/unit/TargetingMap.spec.js

```diff
@@ -0,0 +1,111 @@
+import TargetingMap from '../../src/TargetingMap';
+
+/** @test {TargetingMap} */
+describe('TargetingMap', () => {
+  /** @test {TargetingMap#get} */
+  describe('#get', () => {
+    it('returns empty array if not set', () => {
+      const map = new TargetingMap();
```

Comment:
Nit (minor) - could build a map in before each or use a higher level function to build the test.